### ISSUE TITLE
Bulk install issue with bundled plugins - develop branch

### ISSUE THREAD

**Issue by tanshcreative** [thomasgriffin/TGM-Plugin-Activation#316]
Hello,

Sorry if this has been previously notified and I couldn't find.

<strong>Case 1:</strong>  Bulk install of bundled plugins only

![case-1](https://cloud.githubusercontent.com/assets/12083094/7296878/2c9e54c0-e9e1-11e4-9857-b5600bd3302b.jpg)

<strong>Case 2:</strong>  Bulk install of - Bundled Plugins + Plugins via WP repository 

![case-2](https://cloud.githubusercontent.com/assets/12083094/7296883/32dbf48c-e9e1-11e4-96cf-61bead85b665.jpg)

Thanks, Tansh

**Comment by GaryJones** [comment]
Which version did you try with? The one in the `master` branch as of this morning, the `develop` branch, or a version previous to today?

**Comment by tanshcreative** [comment]
develop branch and the one from here : https://github.com/thomasgriffin/TGM-Plugin-Activation/releases/tag/2.4.1

Both of above have same issue. 
Can confirm this, as two other users has same issue.

**Comment by GaryJones** [comment]
Working for me on the develop branch. Can you double-check that the `source` values are correct when registering the plugins?

**Comment by tanshcreative** [comment]
Yes. 
Note: Individual bundled plugin install works fine.

**Comment by GaryJones** [comment]
Can you email me the slider or visual composer plugin (github at gamajo.com) and I'll try it here.

**Comment by tanshcreative** [comment]
I will mail with any free plugin bundled (hope that is fine ;) ) , as not sure if I can share those.
Will create a test theme with default WP theme and mail.

**Comment by tanshcreative** [comment]
@ GaryJones Just sent you theme. Thanks.

**Comment by GaryJones** [comment]
Thanks for the email.

Worked first time here:

![screenshot 2015-04-23 14 43 39](https://cloud.githubusercontent.com/assets/88371/7298404/4bc0ed14-e9c7-11e4-9354-0550bcac62a4.png)

**Comment by tanshcreative** [comment]
Hmm... just tested..working like a charm online.
This issue is while working with wampserver. Should I close this issue then?

**Comment by jrfnl** [comment]
I just ran into a similar issue myself - installation of bundled zips not working and found that it was a path issue with how I've got my local test environment set up.

Double check this line in your setup for the bundled zips:
```php
'source'             => get_stylesheet_directory() . '/lib/plugins/tgm-example-plugin.zip',
```

If I tried to install just a single one of the bundled ones - I could see the path WP was trying to use and it became obvious where things were going wrong.

If that solves it for you, feel free to close the issue.

**Comment by tanshcreative** [comment]
Thanks.

**Comment by GaryJones** [comment]
If it's WampServer, might there be a discrepency between `/` and `\` @jrfnl?

**Comment by jrfnl** [comment]
Actually no, something else is going on - has to do with the `esc_url`'s we added in 2.4.1, similar to the `http://repo` issue. Am debugging this now.

**Comment by jrfnl** [comment]
I believe I've solved this now - see PR #321

**Comment by tanshcreative** [comment]
Yes. You did it! Working fine now.
